### ISSUE TITLE
Add was_pressed method for button presses?

### ISSUE THREAD

**Issue by seantibor** [adafruit/Adafruit_CircuitPython_CircuitPlayground#50]
The microbit version of micropython has two different methods for accessing button state. The first, is_pressed(), works much like the implementation in the cpx library, and is useful for getting the current button state. This is documented on [Read The Docs](https://microbit-micropython.readthedocs.io/en/latest/button.html)

The second method, was_pressed(), returns True if the button has been pressed since the last time the method was called or since startup if this is the first time the method has been called.

The second method is useful for beginner programmers to detect button presses while the device may have been sleeping or performing other tasks. This also simplifies debounce and sustained presses. 

Is it possible to add a method like this to the cpx library to ease the use of buttons?

**Comment by ladyada** [comment]
this is a handy idea, @kattni if you'd like to implement its only a lilt bit of bitwise math

**Comment by deshipu** [comment]
I don't think it's just math, because you need either a pin change interrupt or background polling to get the button presses when you are not explicitly checking them. The micro:bit does it with the latter, just like the ``gamepad`` module does.

**Comment by deshipu** [comment]
This is exactly what I wrote the ``gamepad`` for.

**Comment by ladyada** [comment]
ok! wanna check if its frozen in, and if not, try adding?

**Comment by deshipu** [comment]
It's compiled by default on the CPX. Should we have a helper in the ``express`` module for it? I guess I could make an object called ``was_pressed`` with ``button_a`` and ``button_b`` properties?

**Comment by ladyada** [comment]
yah it would be integrated into this library - wanna submit a PR?

**Comment by deshipu** [comment]
Turns out it's tricky to do separate properties for separate buttons, because checking either of them would clear the state, so instead I added just a single property ``were_pressed`` that returns a set. Without docs or examples for now, just to see if that is what we want: #51

**Comment by deshipu** [comment]
You would use it like this:

```python

pressed = cpx.were_pressed
if 'A' in pressed:
    ...
if 'B' in pressed:
    ...
```

**Comment by dherrada** [comment]
Fixed in #72. Will merge within an hour or two
